### PR TITLE
Version 3.0.3 release.

### PR DESCRIPTION
**Pull request by carltongibson**

### CODE REVIEW

File: docs/releases/3.0.3.rst [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+potential leakage of session identifiers and other sensitive data.
+
+This issue affects Channels 3.0.x before 3.0.3, and is resolved in Channels
+3.0.3.
+
+Users of ``ProtocolTypeRouter`` not explicitly specifying the handler for the
+``'http'`` key, or those explicitly using ``channels.http.AsgiHandler``, likely
+to support Django v2.2, are affected and should update immediately.
+
+Note that both not explicitly specifying the handler for the ``'http'`` key
```
Comment (smithdc1) [inline docs/releases/3.0.3.rst:22]: 'both not' is a bit hard to read. Could remove the word 'both'?

File: docs/releases/3.0.3.rst [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+and using ``channels.http.AsgiHandler`` are deprecated, and will raise a
+warning, from Channels v3.0
```
Comment (smithdc1) [inline docs/releases/3.0.3.rst:24]: Add patch number here? Is this 3.0.0 (I'm assuming it is)?

File: docs/releases/3.0.3.rst [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+    # Import other Channels classes and consumers here.
+    from channels.routing import ProtocolTypeRouter, URLRouter
+
+    application = ProtocolTypeRouter({
+        # Explicitly set 'http' key using Django's ASGI application.
+        "http": django_asgi_app,
+
+        # Other protocols here.
+        ),
```
Comment (smithdc1) [inline docs/releases/3.0.3.rst:53]: Think this line can be removed?

File: docs/releases/3.0.3.rst (hunk 1 above)
Comment (carltongibson) [reply docs/releases/3.0.3.rst]: Adjusted.

File: docs/releases/3.0.3.rst (hunk 3 above)
Comment (smithdc1) [reply docs/releases/3.0.3.rst:51]: Sorry. I was trying to refer to the closing bracket here. That is the `)`

**Pull request merged**